internal/service/advancedcluster/fw_data_source_advanced_cluster.go

```diff
@@ -0,0 +1,428 @@
+package advancedcluster
```

Comment:
This file will be renamed to `data_source_advanced_cluster.go` after the review, currently named with `fw_*` so PR review is easier and cleaner.